applications_superstaq/user_config.py

```diff
@@ -52,3 +53,38 @@ def aqt_upload_configs(self, pulses_file_path: str, variables_file_path: str) ->
         )
 
         return json_dict
+
+    def aqt_get_configs(self, pulses_file_path: str, variables_file_path: str) -> None:
+        """Writes AQT configs from the AQT system onto the given file paths.
+
+        Args:
+            pulses_file_path: where to write the pulse configurations
+            variables_file_path: where to write the variables configurations
+        Returns:
+            None
+        """
+        pulses_file_exists = os.path.exists(pulses_file_path)
+        variables_file_exists = os.path.exists(variables_file_path)
+
+        if pulses_file_exists and not variables_file_exists:
+            raise ValueError(
+                f"{pulses_file_path} exists as a pulse file. "
```

Comment:
nit
```suggestion
                f"{pulses_file_path} exists. "
```
(e.g. if i write `aqt_get_configs("Documents/my-grocery-list", ...)` it might exist but not "as a pulse file")

(ditto below)

Reply:
Done.